sycl/source/detail/pi.cpp

```diff
@@ -53,6 +55,16 @@ namespace pi {
 
 bool XPTIInitDone = false;
 
+void contextSetExtendedDeleter(cl::sycl::context context,
+                               pi_context_extended_deleter func,
+                               void *user_data) {
+  auto impl = getSyclObjImpl(context);
+  auto contextHandle = reinterpret_cast<pi_context>(context.get());
```

Comment:
Please, use `getHandleRef`. get increments reference count(piRetainContext).

Reply:
Could you please resolve this one?
```suggestion
  auto contextHandle = reinterpret_cast<pi_context>(impl->getHandleRef());
```